### PR TITLE
chore(wal): make TableNameRegistry case-insensitive

### PR DESCRIPTION
**Pull request by eugenels**
This PR changes the case sensitivity of the TableNameRegistry keys.

This change, combined with TableToken, provides case insensitivity for table names.

### CODE REVIEW

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 870 / 941 (92.45%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/PropServerConfiguration.java|0|1|00.00%|
|:large_blue_circle:|io/questdb/cairo/wal/WalUtils.java|0|1|00.00%|
|:large_blue_circle:|io/questdb/griffin/ExpressionParser.java|0|1|00.00%|
|:large_blue_circle:|io/questdb/ServerMain.java|0|2|00.00%|
|:large_blue_circle:|io/questdb/std/AbstractCharSequenceHashSet.java|0|1|00.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/DefaultLineTcpReceiverConfiguration.java|0|1|00.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpUtils.java|0|1|00.00%|
|:large_blue_circle:|io/questdb/log/LogFactory.java|1|2|50.00%|
|:large_blue_circle:|io/questdb/std/ConcurrentHashMap.java|46|77|59.74%|
|:large_blue_circle:|io/questdb/cairo/wal/WalWriter.java|184|200|92.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpMeasurementEvent.java|23|25|92.00%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|28|30|93.33%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpMeasurementScheduler.java|239|250|95.60%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/UpdateOperation.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpEventBuffer.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/std/IntList.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/cairo/vm/api/MemoryCARW.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/seq/SequencerMetadataService.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/AbstractOperation.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/AlterOperationBuilder.java|35|35|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/WalWriterEvents.java|22|22|100.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/TableUpdateDetails.java|41|41|100.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpWriterJob.java|12|12|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/seq/TableSequencerImpl.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/std/Chars.java|28|28|100.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpNetworkIOJob.java|11|11|100.00%|
|:large_blue_circle:|io/questdb/mp/Job.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/std/str/FloatingDirectCharSink.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cairo/TableNameRegistryRW.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/CompiledQueryImpl.java|7|7|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/ApplyWal2TableJob.java|25|25|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/seq/TableSequencerAPI.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/std/DirectByteCharSequenceIntHashMap.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/cairo/vm/MemoryPARWImpl.java|17|17|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlCompiler.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/WriterRowUtils.java|12|12|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/seq/TableTransactionLog.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/cairo/TableNameRegistryRO.java|6|6|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/AlterOperation.java|91|91|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/OperationCompiler.java|10|10|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 869 / 942 (92.25%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/PropServerConfiguration.java|0|1|00.00%|
|:large_blue_circle:|io/questdb/cairo/wal/WalUtils.java|0|1|00.00%|
|:large_blue_circle:|io/questdb/griffin/ExpressionParser.java|0|1|00.00%|
|:large_blue_circle:|io/questdb/ServerMain.java|0|2|00.00%|
|:large_blue_circle:|io/questdb/std/AbstractCharSequenceHashSet.java|0|1|00.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/DefaultLineTcpReceiverConfiguration.java|0|1|00.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpUtils.java|0|1|00.00%|
|:large_blue_circle:|io/questdb/log/LogFactory.java|1|2|50.00%|
|:large_blue_circle:|io/questdb/std/ConcurrentHashMap.java|45|78|57.69%|
|:large_blue_circle:|io/questdb/cairo/wal/WalWriter.java|184|200|92.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpMeasurementEvent.java|23|25|92.00%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|28|30|93.33%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpMeasurementScheduler.java|239|250|95.60%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/UpdateOperation.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpEventBuffer.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/std/IntList.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/cairo/vm/api/MemoryCARW.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/seq/SequencerMetadataService.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/AbstractOperation.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/AlterOperationBuilder.java|35|35|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/WalWriterEvents.java|22|22|100.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/TableUpdateDetails.java|41|41|100.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpWriterJob.java|12|12|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/seq/TableSequencerImpl.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/std/Chars.java|28|28|100.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpNetworkIOJob.java|11|11|100.00%|
|:large_blue_circle:|io/questdb/mp/Job.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/std/str/FloatingDirectCharSink.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cairo/TableNameRegistryRW.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/CompiledQueryImpl.java|7|7|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/ApplyWal2TableJob.java|25|25|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/seq/TableSequencerAPI.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/std/DirectByteCharSequenceIntHashMap.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/cairo/vm/MemoryPARWImpl.java|17|17|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlCompiler.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/WriterRowUtils.java|12|12|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/seq/TableTransactionLog.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/cairo/TableNameRegistryRO.java|6|6|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/AlterOperation.java|91|91|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/OperationCompiler.java|10|10|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 881 / 942 (93.52%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/PropServerConfiguration.java|0|1|00.00%|
|:large_blue_circle:|io/questdb/cairo/wal/WalUtils.java|0|1|00.00%|
|:large_blue_circle:|io/questdb/griffin/ExpressionParser.java|0|1|00.00%|
|:large_blue_circle:|io/questdb/ServerMain.java|0|2|00.00%|
|:large_blue_circle:|io/questdb/std/AbstractCharSequenceHashSet.java|0|1|00.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/DefaultLineTcpReceiverConfiguration.java|0|1|00.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpUtils.java|0|1|00.00%|
|:large_blue_circle:|io/questdb/log/LogFactory.java|1|2|50.00%|
|:large_blue_circle:|io/questdb/std/ConcurrentHashMap.java|57|78|73.08%|
|:large_blue_circle:|io/questdb/cairo/wal/WalWriter.java|184|200|92.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpMeasurementEvent.java|23|25|92.00%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|28|30|93.33%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpMeasurementScheduler.java|239|250|95.60%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/UpdateOperation.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpEventBuffer.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/std/IntList.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/cairo/vm/api/MemoryCARW.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/seq/SequencerMetadataService.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/AbstractOperation.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/AlterOperationBuilder.java|35|35|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/WalWriterEvents.java|22|22|100.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/TableUpdateDetails.java|41|41|100.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpWriterJob.java|12|12|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/seq/TableSequencerImpl.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/std/Chars.java|28|28|100.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpNetworkIOJob.java|11|11|100.00%|
|:large_blue_circle:|io/questdb/mp/Job.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/std/str/FloatingDirectCharSink.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cairo/TableNameRegistryRW.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/CompiledQueryImpl.java|7|7|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/ApplyWal2TableJob.java|25|25|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/seq/TableSequencerAPI.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/std/DirectByteCharSequenceIntHashMap.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/cairo/vm/MemoryPARWImpl.java|17|17|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlCompiler.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/WriterRowUtils.java|12|12|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/seq/TableTransactionLog.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/cairo/TableNameRegistryRO.java|6|6|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/AlterOperation.java|91|91|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/OperationCompiler.java|10|10|100.00%|

File: core/src/main/java/io/questdb/cairo/TableNameRegistryRW.java [hunk 1]
```diff
@@ -30,8 +30,8 @@
 import java.util.Map;
 
 public class TableNameRegistryRW extends AbstractTableNameRegistry {
-    private final Map<CharSequence, TableToken> nameTableTokenMap = new ConcurrentHashMap<>();
-    private final Map<CharSequence, ReverseTableMapItem> reverseTableNameTokenMap = new ConcurrentHashMap<>();
+    private final ConcurrentHashMap<TableToken> nameTableTokenMap = new ConcurrentHashMap<>(false);
+    private final ConcurrentHashMap<ReverseTableMapItem> reverseTableNameTokenMap = new ConcurrentHashMap<>(false);
```
Comment (ideoma) [inline core/src/main/java/io/questdb/cairo/TableNameRegistryRW.java:34]: I think reverse map can be left case sensitive.

**Review by ideoma — COMMENTED** [review]
This PR includes all the ILP changes, not only case insensitive business. Should be merged / reviewed after WAL ILP changes done

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 881 / 942 (93.52%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/PropServerConfiguration.java|0|1|00.00%|
|:large_blue_circle:|io/questdb/cairo/wal/WalUtils.java|0|1|00.00%|
|:large_blue_circle:|io/questdb/griffin/ExpressionParser.java|0|1|00.00%|
|:large_blue_circle:|io/questdb/ServerMain.java|0|2|00.00%|
|:large_blue_circle:|io/questdb/std/AbstractCharSequenceHashSet.java|0|1|00.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/DefaultLineTcpReceiverConfiguration.java|0|1|00.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpUtils.java|0|1|00.00%|
|:large_blue_circle:|io/questdb/log/LogFactory.java|1|2|50.00%|
|:large_blue_circle:|io/questdb/std/ConcurrentHashMap.java|57|78|73.08%|
|:large_blue_circle:|io/questdb/cairo/wal/WalWriter.java|184|200|92.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpMeasurementEvent.java|23|25|92.00%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|28|30|93.33%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpMeasurementScheduler.java|239|250|95.60%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/UpdateOperation.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpEventBuffer.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/std/IntList.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/cairo/vm/api/MemoryCARW.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/seq/SequencerMetadataService.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/AbstractOperation.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/AlterOperationBuilder.java|35|35|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/WalWriterEvents.java|22|22|100.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/TableUpdateDetails.java|41|41|100.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpWriterJob.java|12|12|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/seq/TableSequencerImpl.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/std/Chars.java|28|28|100.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpNetworkIOJob.java|11|11|100.00%|
|:large_blue_circle:|io/questdb/mp/Job.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/std/str/FloatingDirectCharSink.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cairo/TableNameRegistryRW.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/CompiledQueryImpl.java|7|7|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/ApplyWal2TableJob.java|25|25|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/seq/TableSequencerAPI.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/std/DirectByteCharSequenceIntHashMap.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/cairo/vm/MemoryPARWImpl.java|17|17|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlCompiler.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/WriterRowUtils.java|12|12|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/seq/TableTransactionLog.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/cairo/TableNameRegistryRO.java|6|6|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/AlterOperation.java|91|91|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/OperationCompiler.java|10|10|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 881 / 942 (93.52%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/PropServerConfiguration.java|0|1|00.00%|
|:large_blue_circle:|io/questdb/cairo/wal/WalUtils.java|0|1|00.00%|
|:large_blue_circle:|io/questdb/griffin/ExpressionParser.java|0|1|00.00%|
|:large_blue_circle:|io/questdb/ServerMain.java|0|2|00.00%|
|:large_blue_circle:|io/questdb/std/AbstractCharSequenceHashSet.java|0|1|00.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/DefaultLineTcpReceiverConfiguration.java|0|1|00.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpUtils.java|0|1|00.00%|
|:large_blue_circle:|io/questdb/log/LogFactory.java|1|2|50.00%|
|:large_blue_circle:|io/questdb/std/ConcurrentHashMap.java|57|78|73.08%|
|:large_blue_circle:|io/questdb/cairo/wal/WalWriter.java|184|200|92.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpMeasurementEvent.java|23|25|92.00%|
|:large_blue_circle:|io/questdb/cairo/TableWriter.java|28|30|93.33%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpMeasurementScheduler.java|239|250|95.60%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/UpdateOperation.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpEventBuffer.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/std/IntList.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/cairo/vm/api/MemoryCARW.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/seq/SequencerMetadataService.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/AbstractOperation.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/AlterOperationBuilder.java|35|35|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/WalWriterEvents.java|22|22|100.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/TableUpdateDetails.java|41|41|100.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpWriterJob.java|12|12|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/seq/TableSequencerImpl.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/std/Chars.java|28|28|100.00%|
|:large_blue_circle:|io/questdb/cutlass/line/tcp/LineTcpNetworkIOJob.java|11|11|100.00%|
|:large_blue_circle:|io/questdb/mp/Job.java|2|2|100.00%|
|:large_blue_circle:|io/questdb/std/str/FloatingDirectCharSink.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cairo/TableNameRegistryRW.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/griffin/CompiledQueryImpl.java|7|7|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/ApplyWal2TableJob.java|25|25|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/seq/TableSequencerAPI.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/std/DirectByteCharSequenceIntHashMap.java|4|4|100.00%|
|:large_blue_circle:|io/questdb/cairo/vm/MemoryPARWImpl.java|17|17|100.00%|
|:large_blue_circle:|io/questdb/griffin/SqlCompiler.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/WriterRowUtils.java|12|12|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/seq/TableTransactionLog.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/cairo/TableNameRegistryRO.java|6|6|100.00%|
|:large_blue_circle:|io/questdb/griffin/engine/ops/AlterOperation.java|91|91|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/OperationCompiler.java|10|10|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 67 / 88 (76.14%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/std/ConcurrentHashMap.java|57|78|73.08%|
|:large_blue_circle:|io/questdb/cairo/TableNameRegistryRW.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/seq/TableSequencerAPI.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cairo/TableNameRegistryRO.java|6|6|100.00%|

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 67 / 88 (76.14%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/std/ConcurrentHashMap.java|57|78|73.08%|
|:large_blue_circle:|io/questdb/cairo/TableNameRegistryRW.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/seq/TableSequencerAPI.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cairo/TableNameRegistryRO.java|6|6|100.00%|

**Review by ideoma — COMMENTED** [review]
It's a good solid change. However for piece of mind can you do a JMH bench of put() / get() of case insensitive flavour before and after the change?

**Comment by eugenels** [comment]
> It's a good solid change. However for piece of mind can you do a JMH bench of put() / get() of case insensitive flavour before and after the change?

```
Benchmark                                          Mode  Cnt  Score   Error  Units
ConcurrentHashMapBenchmark.testCaseInsensitiveMap  avgt    3  0.765 ± 0.480  us/op
ConcurrentHashMapBenchmark.testCaseSensitiveMap    avgt    3  0.704 ± 0.239  us/op
ConcurrentHashMapBenchmark.testOriginalHashMap     avgt    3  0.689 ± 0.035  us/op
```

**Review by ideoma — APPROVED** [review]

**Comment by ideoma** [comment]
#### [PR Coverage check]

:heart_eyes: **pass** : 67 / 88 (76.14%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|io/questdb/std/ConcurrentHashMap.java|57|78|73.08%|
|:large_blue_circle:|io/questdb/cairo/TableNameRegistryRW.java|3|3|100.00%|
|:large_blue_circle:|io/questdb/cairo/wal/seq/TableSequencerAPI.java|1|1|100.00%|
|:large_blue_circle:|io/questdb/cairo/TableNameRegistryRO.java|6|6|100.00%|

**Pull request merged**